src/Data/Skills/act_int.lua

```diff
@@ -7694,6 +7694,9 @@ skills["FrostBombPlayer"] = {
 				area = true,
 				duration = true,
 			},
+			baseMods = {
+				mod("ColdExposure", "BASE", -20, 0, 0, { type = "GlobalEffect", effectType = "Debuff" }),
```

Comment:
This should not be a base mod, it should be a statmap that grabs the cold exposure value from `skill_cold_exposure_magnitude`